### PR TITLE
Dev raro 473

### PR DESCRIPTION
**Pull request by msingh05**

### CODE REVIEW

File: src/main/webapp/WEB-INF/themes/desktop/resource/js/pages/aperta_article.js [hunk 1]
```diff
@@ -39,6 +39,12 @@ $(document).ready(function () {
     $("#authors").addClass("section-box");
     $("#authors").insertAfter(h2);
 
+    $(".author-corresponding").each(function(index, item) {
+        var html = $(item).html();
```
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/resource/js/pages/aperta_article.js]: I think javascript is not the right place for display details such as these. Let's put them in the css layer. it's also a less expensive operation.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/vendor/kube/components/_typography.scss [hunk 2]
```diff
@@ -20,7 +20,7 @@ a:hover {
 // Headings
 h1.title, h1, h2, h3, h4, h5, h6 {
 	font-family: $headings-font-family;
-	font-weight: bold;
+	font-weight: 600;
```
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/vendor/kube/components/_typography.scss]: we should probably just overwrite this as we agreed, let me check in the stuff i did when we talked last week.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss [hunk 3]
```diff
@@ -16,12 +16,13 @@
...
     background-size: 32px 24px;
     background-repeat: no-repeat;
-
+    margin-bottom: 4px;
   }
   .icon+* {
     align-self: center;
     text-transform: uppercase;
     font-size: $font-size-smaller;
+    font-weight: 600;
```
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss]: let's use @extend .strong which we altered to be font-weight bold. im worried about leaving these littered about to cleanup

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss [hunk 4]
```diff
@@ -42,8 +11,8 @@
     border-top: solid 1px $color-border;
     margin-bottom: 0;
     line-height: 30px;
-    padding-top: 8px;
-    padding-bottom: 8px;
+    padding-top: 3px;
```
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss]: should we create an super tiny padding? seems like we use 3 a lot, you could even keep it confined to this component.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/vendor/kube/components/_typography.scss (hunk 2 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/vendor/kube/components/_typography.scss]: let me know if you are handling it.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss (hunk 3 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss]: let me know if you are handling it. or if you can paste the syntax that will be great.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss (hunk 4 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss]: done.

File: src/main/webapp/WEB-INF/themes/desktop/resource/js/pages/aperta_article.js (hunk 1 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/resource/js/pages/aperta_article.js]: done.

**Comment by msingh05** [comment]
do you know how to fix log message as I accidentally put other ticket number.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/vendor/kube/components/_typography.scss (hunk 2 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/vendor/kube/components/_typography.scss]: done.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss (hunk 3 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss]: done.

**Pull request merged**